chaincode-labs/chaincode-podcast/2021-05-13-Matt-Corallo-and-Lightning-Development-Kit-Episode-13.md

```diff
@@ -0,0 +1,127 @@
+---
+title: Matt Corallo and Lightning Development Kit - Episode 13
+transcript_by: Whisper AI & PyAnnote
+categories: ['podcast']
+tag: ['Starting Rust-Lightning', 'Language bindings challenges FFI', 'Interoperability of Lightning', 'Zero-value invoices', 'Keysend/push payments/spontaneous payments', 'What is the LDK stack?', 'Anchor Outputs Child pays for parent (CPFP)', 'Who tracks the onchain state when using LND?', 'LDK tracks the channel commitment transaction, how is that done? Compact block filters', 'Contrasting multiple implementations working from a spec in Lighting vs. no spec with one dominant reference implementation in Bitcoin The Lightning Spec', "What's the state of the Lightning Network? Have we moved beyond #reckless? Denial of Service (DoS) vulnerabilities Channel Jamming", 'Eltoo and the punishment dynamic', 'Will the network trend to trusted relationships and lend itself to KYC?', 'FATF updates its guidance on Virtual Asset Service Provider (VASP)', 'Where LDK goes from here?']
+---
+
+Chaincode Labs podcast: Matt Corallo and Lightning Development Kit - Episode 13
```

Comment:
for comparison: https://btctranscripts.com/chaincode-labs/chaincode-podcast/2021-05-12-matt-corallo-ldk/